### PR TITLE
Alchemy

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

File: src/pages/developers/services/alchemy.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  jsonrpc: "2.0",
+  id: 1,
+  method: "eth_blockNumber",
+  params: [],
+};
+
+axios
+  .post(url, payload)
+  .then((response) => {
+    console.log("Block Number:", response.data.result);
```
Comment (lucas-janon) [inline src/pages/developers/services/alchemy.mdx:93]: Nit:
```suggestion
    console.log("Block Number: ", response.data.result);
```

**Review by lucas-janon — APPROVED** [review]
LGTM

File: src/pages/developers/services/alchemy.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+```
+node index.js
+```
+
+You should a see the current block number on ZetaChain outputted to your
+console:
+
+```
+Block Number: 0x37eca6
```
Comment (andresaiello) [inline src/pages/developers/services/alchemy.mdx:110]: maybe should cast it to int in line 93 so user can validate it's the current block number. But if we just want to say `if there's an output it's fine` then ok

**Review by andresaiello — APPROVED** [review]

File: src/pages/developers/services/alchemy.mdx (hunk 2 above)
Comment (fadeev) [reply src/pages/developers/services/alchemy.mdx]: Good suggestion! 505a83c6b52efe9078aac2203ad41066d52ac832

**Pull request merged**